### DISCUSSION TITLE
Magnetic mounting

### DISCUSSION THREAD

**Discussion by RIMSKO** [PricelessToolkit/MailBoxGuard#46] [General]
First of all, a big thank you for a great product! The range is impressive it works great from our underground garage through concrete and steel up to the second floor apartment. Is it okay to scale up the height of the bottom part of the case when I print it and put two magnets under the battery or does it interfere or destroy the function?

**Comment by PricelessToolkit** [comment]
Thank you for supporting my work, you can use magnets if you have not added a reed switch on the PCB.  For case Step files are available "https://github.com/PricelessToolkit/MailBoxGuard/tree/main/3D/Step" You can modify them.

**Reply by RIMSKO** [reply]
Thanks for the quick reply, I printed a back piece with an insert for 2 pcs 10x5 mm magnets, scaled up the height a bit, works perfectly.

![20240228_180309](https://github.com/PricelessToolkit/MailBoxGuard/assets/94564281/1abbc2be-0dcb-4a77-9b03-037db6f5d3a1)

**Reply by PricelessToolkit** [reply]
Nice, i hope you will share it.

**Reply by RIMSKO** [reply]
Made a new backpiece 5 mm higher than original and holder for 2 pcs. 10x5 mm magnets also made a "sensitivity limiter" as I have a large opening in my box so it was sometimes triggered when someone was near the opening. Tried to upload the files here if anyone is interested but was unable to upload the STL or STEP.

![20240229_224330](https://github.com/PricelessToolkit/MailBoxGuard/assets/94564281/228061c7-11b0-4273-93f3-cc60d70bb116)
![20240229_224439](https://github.com/PricelessToolkit/MailBoxGuard/assets/94564281/4a475699-76ed-44cd-add3-0b5d3c7b6523)
![20240301_160847](https://github.com/PricelessToolkit/MailBoxGuard/assets/94564281/b074c4d2-8392-437f-8ead-cb521b9141cd)

**Reply by PricelessToolkit** [reply]
I made an Upload link for you, https://share.iot-dev.me/g/HRBoMCyYTmQZoyiq after I will put 3D files on GitHub.
Thanks

**Comment by PricelessToolkit** [comment]
Thanks for your contribution!!  I uploaded files to GitHub.

**Comment by APT256** [comment]
What size magnets are recommended for the magnetic case?

**Reply by PricelessToolkit** [reply]
10x5 mm magnets

**Reply by APT256** [reply]
Thanks!

**Comment by PricelessToolkit** [comment]
I have made a new magnetic mount that is a bit slimmer. The magnet will be in direct contact with the metal, providing a much stronger attraction "MailBoxGuard_Full_Size_Back_433_868_915_Magnetic_Open_12x2mm.STL"

**Reply by RIMSKO** [reply]
Nice!
